sql/core/src/main/scala/org/apache/spark/sql/GroupedData.scala

```diff
@@ -26,42 +26,14 @@ import org.apache.spark.sql.catalyst.expressions._
 import org.apache.spark.sql.catalyst.plans.logical.{Rollup, Cube, Aggregate}
 import org.apache.spark.sql.types.NumericType
 
-/**
- * Companion object for GroupedData
- */
-private[sql] object GroupedData {
```

Comment:
I moved this to the end of the file, since it is internal. It would be better for users if they see the non-internal class when they open this file.